app/models/setting.js

```diff
@@ -11,7 +11,7 @@ export default Model.extend(ValidationEngine, {
     logo: attr('string'),
     coverImage: attr('string'),
     icon: attr('string'),
-    defaultLang: attr('string'),
+    default_lang: attr('string'),
```

Comment:
This should remain in camelCase format (e.g. what it was before). On the server side we use the underscore as a separator, but on the client side we use camelCase unless it's absolutely necessary.